Sources/Articles/Documentation.docc/2025/jwt-kit/jwt-kit.md

```diff
@@ -0,0 +1,198 @@
+# Getting Started with Vapor's JWTKit 5: A Practical Introduction to JSON Web Tokens in Swift
```

Comment:
This title should be shorter. Either the leading part of the trailing part. I think the latter part is more searchable for people coming into the Swift ecosystem.

Reply:
Updated!